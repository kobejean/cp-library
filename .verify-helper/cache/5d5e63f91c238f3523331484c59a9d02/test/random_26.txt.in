98
548214211 28405164 843916374
6454280 680489 793110117
186816437 127637 605742012
878038277 235680674 929309590
704274661 161801782 112187806
352181042 300305001 344770179
309498199 265706021 775674913
793110196 115041110 203490771
94111651 22011833 786933379
323172813 299171378 672695402
728531399 367412115 845870942
615789608 227310752 137760783
346688737 248804007 375678586
480562734 123712729 880518515
91242391 32054485 839624448
66404340 62424894 806412491
250223081 28072876 929091522
153627525 18080777 134566090
352837926 9962760 104994672
917654804 729417280 442356668
396282109 164688270 235850141
473930048 360514860 591849524
711521419 431540557 357307237
495679851 493918070 733484967
700887057 13577005 652081846
263857147 146226760 451947475
388264368 249905933 258591020
536564800 217475490 404733005
768526186 214038712 844267236
106031547 98204408 94674678
876866465 190156759 814100393
759849579 244435629 754424169
273711957 259680776 333355320
703722421 595995527 624099986
412562132 289710552 296633758
375498397 314558385 112069508
270043875 120525819 471266718
278635836 212675170 421814662
114555748 7882675 370824554
695513923 364486778 943646200
867147529 729678444 256443085
549706370 314692590 569338336
989520785 85305340 505259240
879829233 829249617 845777748
356872811 269759585 70640607
925676875 442787875 588752450
66994814 58858742 200080434
881018910 750798607 754295426
408540364 21507048 366969426
549681655 432572440 109845535
467980364 465688487 438689208
459838344 5495376 823762546
828922992 591976632 196397160
236773217 200188563 575489166
288213799 141498561 845109017
256552061 71518030 208050340
454851909 80909012 328118625
778225405 574817985 325531330
702584282 460228088 552809304
343760883 296554880 768880872
830238349 667692856 37903722
763316785 180588727 863741626
364105819 121745300 904673935
796794631 25053025 796794631
165058758 129040341 825293790
222246257 103387060 444492514
157887964 78191032 315775928
257573689 192216068 772721067
207948172 142270400 831792688
999999984 419354832 483870960
999999976 720588218 44117646
999999936 867469824 361445760
999999988 76923076 692307684
999999972 527777763 361111101
999999999 648648648 945945945
999999990 989898980 101010100
999999962 797297267 67567565
999999990 666666660 575757570
999999994 857142852 571428568
1000000000 250000000 250000000
999999924 804597640 11494252
999999975 639999984 586666652
999999988 76923076 384615380
999999959 511627886 69767439
999999990 799999992 872727264
1000000000 425000000 25000000
999999924 804597640 597701104
999999972 777777756 472222209
999999975 76923075 948717925
1000000000 218750000 359375000
999999996 333333332 833333330
999999980 21739130 358695645
999999984 580645152 473118272
1000000000 900000000 100000000
999999984 112903224 983870952
999999957 637681132 536231861
1000000000 265625000 796875000
999999944 105263152 644736806
