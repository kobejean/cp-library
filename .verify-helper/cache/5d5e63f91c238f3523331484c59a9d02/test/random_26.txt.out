303765560
43
186816404
58205969
99
352181035
2266300
54
94111584
290669435
90
615789584
175309715
85
91242320
49822806
91
-1
94
229413606
-1
9
101645868
-1
29
4325515
-1
22
384263009
-1
17
253283098
-1
34
206281008
-1
92
139317899
-1
63
10976467
226399655
39
293276332
-1
70
33497398
-1
30
109936252
-1
48
34538417
-1
22
5966265
-1
58
351292130
-1
70
69392418
-1
-1
-1
-1
-1
-1
-1
26
29
64
10
29
12
10
3
11
2
3
17
33
5
7
22
23
2
28
21
30
2
78
3
1
7
10
21
20
